surveyor/collector_statz.go

```diff
@@ -269,7 +277,7 @@ func (sc *StatzCollector) poll() error {
 			key := fmt.Sprintf("%s:%s", stat.Server.Cluster, stat.Server.Host)
```

Comment:
not sure if this should also be updated, `Host` seems wrong since many machines might be set to localhost or 0.0.0.0, ID might be better?

Reply:
Yep, we could get collisions.  Good catch.